frontend/components/shelter/pet/add/Form.tsx

```diff
@@ -77,25 +81,39 @@ const schema: SchemaOf<NewAnimal> = object().shape({
 	animalImages: array().of(imageSchema.required()).required().min(1),
 });
 
+// =============================================================================
+// Helpers
+// =============================================================================
+const dateToDateString = (date: Date | null) => {
+	if (date) {
+		return dayjs(date).format("YYYY-MM-DD");
+	}
+};
+
+// =============================================================================
+// Form
+// =============================================================================
 export const AddPetForm = () => {
 	const handleSubmit = async (
 		values: NewAnimal,
 		actions: FormikHelpers<NewAnimal>,
 	) => {
+		const transformedValues: Shelter.addNewPetApiDomain.requestBody = {
+			...values,
+			dateOfBirth: dateToDateString(values.dateOfBirth),
```

Comment:
How does this work? are dateOfBirth and intakeDate supposed to allow a value of undefined or null?

Doing it this way would replace `dateOfBirth` with value `undefined` if `value` had any `dateOfBirth`.

Reply:
```
let t = { test: 'test' }
{ ...t, test: undefined }

>>> { test: undefined }
```